spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -84,9 +84,9 @@ def resolve(path, http_verb=:get)
       posts_path(locale: :fr, :i18n_posts => 'puestos').should eql '/fr/messages'
     end
 
-    it 'should work without passing parameters in a hash' do
-      post_path("post-title").should eql "/posts/post-title"
-    end
+    # it 'should work without passing parameters in a hash' do
+    #   post_path("post-title").should eql "/posts/post-title"
```

Comment:
In that case, the test could be removed then? Since the purpose of it was to prove you don't have to pass a hash in... but now it looks like you do.

We could alternatively alter `localizable_url_helper` so that if they just pass in a string, we could build a hash with the `id` key referencing that string value. So it keeps it backwards compatible?